plugins/test.yaml

```diff
@@ -0,0 +1 @@
+testS
```

Comment:
I assume this is the test file used to trigger the deployment: it should not be part of the PR though.

Reply:
Hi @esune i am testing stuff to trigger the workflow